controllers/commentController.js

```diff
@@ -50,4 +51,99 @@ export default class CommentController {
       return res.status(500).send({ message: error, status: 500 });
     }
   }
+
+
+  /**
+   * @description It allows a user to like a specific comment.
+   *
+   * @param  {Object} req - The request object.
+   * @param  {Object} res - The response object.
+   * @returns {Object} - Returns the like created.
+   */
+
+  static async like(req, res) {
+    const { commentId } = req.params;
+    try {
+      const comments = await Comment.findOne({ where: { id: commentId } });
+      if (!comments) {
+        throw new Error("You are liking a non-existing comment");
+      }
+    } catch (error) {
+      if (error.message == "You are liking a non-existing comment") {
```

Comment:
use ```===``` instead of ```==```

Reply:
He should activate eslint locally for automatic checking of those kind of errors.